i18n/en.yml

```diff
@@ -364,8 +364,13 @@ en:
   stocks/fa/earnings: earnings dates and reported EPS
   stocks/fa/fraud: key fraud ratios
   stocks/fa/dupont: detailed breakdown for return on equity
+  stocks/fa/profile: company profile
+  stocks/fa/quote: detailed stock quote information
+  stocks/fa/enterprise: company enterprise value
```

Comment:
Add in the `dcfc` command. The text can be "determine the (historical) discounted cash flow"